dev/com.ibm.ws.microprofile.health.2.0_fat/fat/src/com/ibm/ws/microprofile/health20/fat/MultipleHealthCheckTest.java

```diff
@@ -80,7 +96,23 @@ public void testSuccessReadinessCheck() throws Exception {
 
         JsonObject jsonResponse = getJSONPayload(conReady);
         JsonArray checks = (JsonArray) jsonResponse.get("checks");
-        assertEquals(1, checks.size());
+        assertEquals(2, checks.size());
+        assertEquals(true, checks.getJsonObject(0).getString("name").equals("successful-readiness-check") ||
```

Comment:
Similar as above comment, call a method and pass in the String array of the expected names.